Configuration_adv.hpp

```diff
@@ -445,4 +447,26 @@
 // Note that if you use an app to control OAT, ANY debug output will likely confuse that app.
 // Debug output is useful if you are using Wifi to control the OAT or if you are issuing
 // manual commands via a terminal.
-//
+
+#if defined(OAT_DEBUG_BUILD)
+  // AVR based boards have numbers < 1000
+  #if BOARD < 1000
+    /*
+     * Debugging on the mega2560 using avr-stub dissallows application-code from
+     * using the normal (USB) serial port
+     */
+    // Disable debug output
+    #if defined(DEBUG_LEVEL)
+      #undef DEBUG_LEVEL
```

Comment:
So we don't want logging while debugging? Any particular reason for this? Is Serial unavailable?

[Later]
Ahh, I see that Serial is indeed not available.... which begs the question, how do you make it do the things you want to debug?

Reply:
Yeah `avr-stub` needs total control of the serial port. There is `debug_message()` which prints messages to the GDB console (haven't really looked into it too much though) and it's also possible to still use the LCD to navigate. I could look into substituting our serial interface with the GDB console in the future

Reply:
If the LCD works, I think it's fine. The Meade layer is not that complex, I think the light handed DEBUG_LEVEL can handle that.